thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -555,3 +605,23 @@ body {
     }
   }
 }
+
+.performance-tooltip {
+  display: inline-block;
+}
+
+.performance-metric {
+  width: 20px;
+  text-align: right;
+  display: inline-block;
+
+  &__subset {
+    display: inline-block;
+    color: app-shade(black, 6);
+    padding-left: 5px;
+
+    &--null {
+      color: app-shade(black, 4);
+    }
+  }
+}
```

Comment:
Let's put this in its own scss file called `performance-tooltip`

Reply:
Ok, I pulled those styles out.